quickstart-serverless-receive-sms/assets/index.html

```diff
@@ -0,0 +1,112 @@
+                  <p>The <b>Edit this application</b> button will take you to the Twilio Console, where you can make edits to and re-deploy any of the examples in this guide.</p>
+                  <p>For example: try editing the <code>/static-response</code> code so that the response is something unique to you, other than "Hello, World!". Save the Function, click <b>Deploy All</b> to deploy your updates to the code, and try sending a new text message to your Twilio number to see the modified response.</p>
+                </section>
+                <section>
+                  <h2>Next steps</h2>
+                  <p>Once you're satisfied with your changes to <code>/static-response</code>, there are several other code samples to experiment with!</p>
+                  <p>Remember, each code sample is identical to those in the quickstart, and you are encouraged to refer back to <a href="https://www.twilio.com/docs/serverless/functions-assets/quickstart/receive-sms"  target="_blank" rel="noopener">the documentation</a> for deeper explanations related to each sample. Each sample also includes helpful comments, to better explain the use for and reasoning behind each Function.</p>
+                  <h3>Modify and test more examples</h3>
+                  <p>To learn from and follow along with the examples in the documentation, follow these steps:</p>
+                  <ol class="steps">
+                    <li>Decide on an example (it's easiest to go in the same order as the documentation does)</li>
+                    <li>Navigate to your <a href="https://www.twilio.com/console/phone-numbers/incoming" target="_blank" rel="noopener">phone number configuration</a>, and click on the phone number that's currently connected to this application.</li>
+                    <li>Scroll down to the configuration titled <b>A Message Comes In</b>. (It should already be configured to handle messages using type <code>Function</code>, and be pointed to your most recently used Function from this application.)</li>
+                    <li>Under <b>Function Path</b>, click the dropdown, and select the path of the example you would like to test out next.</li>
+                    <li>Click <b>Save</b>. Your Phone number will now be connected to the newly selected example.</li>
+                    <li>Send a test message to your phone number, check the response, and modify then re-deploy the code of the responsible Function from the console as necessary.</li>
+                    <li>Repeat these steps until you've tried all the examples, and feel comfortable handling any incoming SMS with Functions!</li>
+                  </ol>
+                </section>
+                <section>
+                    <!-- APP_INFO_V2 -->
+                </section>
+                <section>
+                  <!-- EDIT_CODE -->
+                </section>
```

Comment:
This is going to be redundant since the `<!-- APP_INFO_V2 -->` will already include the edit button.